components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -7831,6 +7832,27 @@ public void saveAsyncApiDefinition(API api, String jsonText) throws APIManagemen
         }
     }
 
+    public void saveAsyncApiDefinition(API api, String jsonText, String organization) throws APIManagementException {
```

Comment:
Let's use the existing saveAsyncApiDefinition() method and pass organization instead

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10621/commits/d5abe94f46e9e929b22ad4e684ae4d9122edc925